docs/reference.rst

```diff
@@ -0,0 +1,102 @@
+Reference
+=========
+
+.. automodule:: ZEO
+    :members:
+    :show-inheritance:
+
+.. automodule:: ZEO.cache
+    :members:
+    :show-inheritance:
+
+.. automodule:: ZEO.ClientStorage
+    :members:
+    :show-inheritance:
+
+.. automodule:: ZEO._compat
```

Comment:
ZEO doesn't really have a published public API, I think. Most of this should *not* be documented. Probably  `ClientStorage.ClientStorage` is the only thing that should be documented.

Reply:
I was not sure what needed to be documented or not. What do you think would be the drawbacks to document other parts?

Reply:
I don't see the need to document internals that can, and do, change arbitrarily. Publishing their documentation implies that they are a stable API meant to be used, and that's not true.

Reply:
I get it. What about `StorageServer`?

Reply:
Here's one (good?) way to determine what should have its API documented: If it's mentioned in the other documentation (i.e., the original README) it should be documented. Otherwise, no.

That document just includes a few things:
- `ZEO.client`
- `ZEO.DB`
- `ZEO.server`
- `ZEO.Exceptions`

Reply:
Ok I just documented those, plus `ClientStorage` as `ZEO.client` is just a shortcut for `ClientStorage`.